### PR TITLE
Shared with action menu label alignment

### LINKED ISSUE #9323: alignment of expiration date for shares
### Steps to reproduce
1. select a resource
2. share the resource with a person or group
3. select the action menu to set expiration date

### Expected behaviour 
label of the option "set expiration date" should be aligned with the other items of the tippy menu

### Actual behaviour
label is not aligned

the same issue also applies to the option "edit expiration date" once an expiration date has been set and it's the same in "Personal" and "Spaces"

Screenshot_Edit_Expiration_Date

### PR DESCRIPTION
**Pull request by koebel**
## Description
Previously in the label of the option "set expiration date" and “edit expiration date” for resources in the Shares sidebar was not in line. The same problem also applied to expiration date of members in Spaces. I removed the spacing between the icons and labels of the options below “set/edit expiration date” so that the design matches with the spacing of the same feature for sharing links. 

## Related Issue
- Fixes #9323

## Motivation and Context
Maintain consistent design in the UI

## How Has This Been Tested?
inspecting HTML code output, manual testing in Chrome 

## Screenshots (if appropriate):

Design Reference for width of spacing
Screenshot_Shares_Add_Expiration_Date_Design_Reference

Shares after modification
Screenshot_Shares_Set_Expiration_Date_after_modification

Spaces after modification
Screenshot_Spaces_Set_Expiration_Date_after_modification

## Types of changes
- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
- [x ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
- [ ] ...

### CODE REVIEW

File: changelog/unreleased/bugfix-label-alignment [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Shared with action menu label alignment
+
+Adjusted offset of alinment of label for shared with action menu option in Sidebar.
```
Comment (phil-davis) [inline changelog/unreleased/bugfix-label-alignment:3]:
```suggestion
Adjusted offset of alignment of label for shared with action menu option in Sidebar.
```

**Comment by koebel** [comment]
@JammingBen: Jan was suggested by Github as reviewer of this PR (I guess because it is design related?), but since it's still pending I've added you as reviewer. I guess you know best who should have a look at this PR.

**Comment by JanAckermann** [comment]
@koebel thats because I reviewed some PRs already 😅

**Review by JammingBen — APPROVED** [review]
Looks good 👍 I committed Phil's suggestion with the fixed typo.

**Comment by JammingBen** [comment]
@koebel Do you still plan to rebase, or should we take care?

**Comment by koebel** [comment]
@JammingBen thanks for the reminder. I just rebased. Locally all tests passed, so I expect that CI will pass in a little moment :)

**Comment by JammingBen** [comment]
Hmm something is still off, the pipeline doesn't get past the first steps. Are you sure you rebased with current master? 😅 Not sure about the merge commit (https://github.com/owncloud/web/pull/9324/commits/4433cedf898df56a7d931bea0d5fdc855a35c7fb), simply `git rebase origin/master` should do the trick.

**Comment by JammingBen** [comment]
Now there is something off completely, it shows 200+ commits 🙈 Don't know what happened here, probably the easiest way is to reset it:

```
git reset --hard origin/master
git cherry-pick 54bdd385bd7b4d19bb081e83e51caacc7e2fd165
git cherry-pick 5aa2d1bb106db1d1e68ebcb257bc1fc808b91462
```

**Comment by koebel** [comment]
> Now there is something off completely, it shows 200+ commits 🙈 Don't know what happened here, probably the easiest way is to reset it:
> 
> ```
> git reset --hard origin/master
> git cherry-pick 54bdd385bd7b4d19bb081e83e51caacc7e2fd165
> git cherry-pick 5aa2d1bb106db1d1e68ebcb257bc1fc808b91462
> ```

thanks, it seems like it worked now

**Comment by JammingBen** [comment]
There is still something wrong:

image

It also still shows [your merge commit](https://github.com/owncloud/web/pull/9324/commits/06ccbeb2b4b04a446f1bd1e87bf5c9df01f32d74) which shouldn't be there anymore. Are you sure you ran the git commands correctly?

**Comment by koebel** [comment]
> There is still something wrong:
> 
> image
> It also still shows [your merge commit](https://github.com/owncloud/web/pull/9324/commits/06ccbeb2b4b04a446f1bd1e87bf5c9df01f32d74) which shouldn't be there anymore. Are you sure you ran the git commands correctly?

@JammingBen Thanks for your feedback, I don't know what exactly went wrong. There must have been a problem with the rebasing at some point because it shows all (or at least some) of the commits made by others since I created this PR several weeks ago upto now. My suggestion is to close this thread without merging and replicate the minor code changes in a new branch and create a new PR.  I assume this takes less time than trying to reverse engineer what went wrong. Is this approach ok?

**Comment by koebel** [comment]
this PR is replaced by https://github.com/owncloud/web/pull/9529

**Pull request closed without merging**